RELEASE_NOTES.md

```diff
@@ -6,6 +6,12 @@ cspell:words pubspec erickzanardo xcframeworks Cupertino codesign codecov rkisha
 
 This section contains past updates we've sent to customers.
 
+## 1.4.5 (October 23, 2024)
+
+- 🧑‍🔧 Fix bug when patching with `--split-debug-info` on iOS
+  - `shorebird release` and `shorebird patch` now have first party support for `--split-debug-info`
```

Comment:
Do we handle it on either side of the `--`?

Reply:
Yup